code/modules/mob/living/carbon/slime/items.dm

```diff
@@ -25,6 +25,7 @@
 		uses *= 3
 		enhanced = TRUE
 		qdel(O)
+	..()
```

Comment:
```suggestion
		qdel(O)
		return TRUE
	. = ..()
```
/attackby should return a value if it has been 'caught' properly
(not something wrong with your PR, just suggesting to add it while you're there)